include/mrdocs/Metadata/Javadoc.hpp

```diff
@@ -620,8 +698,11 @@ class MRDOCS_DECL
 
     /** Return the brief, or nullptr if there is none.
     */
-    doc::Paragraph const*
-    brief() const noexcept;
+    const doc::Paragraph*
```

Comment:
Vinnie strongly prefers `T const*` over `const T*` and has given me a hard time because of that in the past.